docs(observability): cockpit add python example to help to call Scaleway Loki API (#1213)
scaleway/docs-content: observability/cockpit/how-to/send-metrics-logs-to-cockpit.mdx

     - The Remote Write endpoint to push your logs is the following: `https://logs.prd.obs.fr-par.scw.cloud/loki/api/v1/push`
    </Message>

<Concept opened>

> ## Example: send Python logs to Scaleway Cockpit on a new logs dashboard

Reviewer: ```suggestion
## Example: send Python logs to your Scaleway Cockpit on a new logs dashboard
```

> First we need [logging_loki](https://pypi.org/project/python-logging-loki/), you can install with `pip install python-logging-loki`

Reviewer: ```suggestion
First we need [logging_loki](https://pypi.org/project/python-logging-loki/). You can install it using the command `pip install python-logging-loki`
```

```
snippet_python_to_scw_cockpit.py

import logging
import logging_loki

handler = logging_loki.LokiHandler(
    url="https://logs.prd.obs.fr-par.scw.cloud/loki/api/v1/push", 
    tags={"job": "logs_from_python"},
    auth=("api_key", "YOUR_COCKPIT_API_KEY_WITH_PUSH_LOGS_AUTHORIZATION"),
    version="1",
)

logger = logging.getLogger("my-first-python-logger")
logger.addHandler(handler)
logger.error(
    "Logging a python error with Scaleway cockpit example", 
    extra={"tags": {"service": "my-service"}},
    )
```

>     <Message type="tip">

Reviewer 2: ```suggestion
<Message type="tip">
```

> On Loki Scaleway API, ou can POST with bearer token authorization, or POST directly https://api_key:YOUR_COCKPIT_API_KEY@logs.prd.obs.fr-par.scw.cloud/loki/api/v1/push

Reviewer 2: ```suggestion
  On Loki Scaleway API, you can POST with bearer token authorization, or POST directly `https://api_key:YOUR_COCKPIT_API_KEY@logs.prd.obs.fr-par.scw.cloud/loki/api/v1/push`.
```

>     </Message>

Reviewer 2: ```suggestion
</Message>
```

> Python logs are send to Cockpit from Scaleway Loki API. Now, we need to retrieve them on Scaleway Cockpit with grafana:

Reviewer: ```suggestion
Python logs are sent to your Cockpit from the Scaleway Loki API. Now, we need to retrieve them on  the Scaleway Cockpit with Grafana:
```

> Python logs are send to Cockpit from Scaleway Loki API. Now, we need to retrieve them on Scaleway Cockpit with grafana :

Reviewer 2: ```suggestion
Python logs are send to Cockpit from Scaleway Loki API. Now, we need to retrieve them on Scaleway Cockpit with grafana:
```

> * We choose a [simple logs dashboard](https://grafana.com/grafana/dashboards/13639-logs-app/) for this exemple on Grafana Dashoards. Grafana Dashboard id 13639

Author: thank you for proofreading @bene2k1

> * We choose a [simple logs dashboard](https://grafana.com/grafana/dashboards/13639-logs-app/) for this exemple on Grafana Dashoards (Grafana Dashboard ID `13639`).

Reviewer: ```suggestion
* We choose a [simple logs dashboard](https://grafana.com/grafana/dashboards/13639-logs-app/) for this example on Grafana Dashoards (Grafana Dashboard ID `13639`).
```

* On Scaleway Cockpit, choose "Dashboards", "Import", "Import via grafana.com" and load url or Grafana Dashboard id directly, here : id 13639

> * On your new Dashboard, you can view log line from Python snippet. Congratulation, no more complexity !

Reviewer: ```suggestion
* On your new dashboard, you can view your log line from Python snippet.
```
